analysis/topeft_run2/analysis_processor.py

```diff
@@ -322,7 +314,7 @@ def process(self, events):
         e_fo = ele[ele.isPres & ele.isLooseE & ele.isFO]
 
         # Attach the lepton SFs to the electron and muons collections
-        AttachElectronSF(e_fo, year=year, looseWP=ele_pres_id_tag) #Run3 ready
+        AttachElectronSF(e_fo, year=year, looseWP="wpLnoiso" if is_run3 else "none") #Run3 ready
```

Comment:
Is there no tag for run 2?

Reply:
for run 3, we don't apply any of the EGM-provided cuts on the preselection. But at the end this depends on how we implement the lepton scale factors, which should come in a separate PR. same answer for the comment below

Reply:
But isn't this passing `None` for run 2 instead of run 3?

Reply:
nice catch, indeed. I'll fix that. I'm  wondering: why wasn't this caught in the yield comparison in the CI?

Reply:
That's a very good question. Could the differences be small enough?